Half/half.cpp

```diff
@@ -54,28 +53,26 @@ using namespace std;
 //-------------------------------------------------------------
 
 HALF_EXPORT const half::uif half::_toFloat[1 << 16] =
-    #include "toFloat.h"
-HALF_EXPORT const unsigned short half::_eLut[1 << 9] =
-    #include "eLut.h"
-
-//-----------------------------------------------
-// Overflow handler for float-to-half conversion;
-// generates a hardware floating-point overflow,
-// which may be trapped by the operating system.
-//-----------------------------------------------
-
-HALF_EXPORT float
-half::overflow ()
+#include "toFloat.h"
```

Comment:
This section is weird. I think the presence of the `#include` is throwing off its analysis of the syntax. Maybe a bracketing `// clang-format off` `// clang-format on` should be used here.